go/test/action.yaml

```diff
@@ -54,7 +58,7 @@ runs:
 
     - name: Run tests
       shell: bash
-      run: go test ./... -v -coverpkg=./... -coverprofile cover.out
+      run: go test ./cmd/... ./pkg/... -v -coverpkg=./cmd/...,./pkg/... -coverprofile cover.out
```

Comment:
What if some modules would have an [internal](https://github.com/golang-standards/project-layout#internal) directory as well?

Reply:
What purpose does this change have?

Reply:
@Decard6 https://github.com/PiwikPRO/PPMS-DomainResolver/pull/60/files
@radekska I think that we should introduce a global convention of putting the go project files only in `pkg` and `cmd` directory in order for it to work. If you plan to use `internal` packages, there's no problem in putting them under `pkg/internal` i suppose.

Reply:
@radekska Hmm, now i've read your link. Should we add the `internal` package to the flags as well?

Reply:
@glothriel not sure how many modules actually use a separate `internal` package, though making test and coverage directories configurable would be a plus

Reply:
I'd say so, we probably want to adjust to `go` standards.

Reply:
I'm going to get back to this on next pet project friday